static/app/components/events/interfaces/sourceMapsDebuggerModal.tsx

```diff
@@ -255,6 +255,10 @@ function SentryWizardCallout({
   projectSlug?: string;
 }) {
   const isSelfHosted = ConfigStore.get('isSelfHosted');
+  const wizardCommand = `npx @sentry/wizard@latest -i sourcemaps${
+    isSelfHosted ? '' : ' --saas'
+  } --org ${orgSlug} --project ${projectSlug}`;
```

Comment:
I noticed that we only moved the code to be in a const. do we want to use `getWizardCommand` here?